site/content/en/docs/Examples/simple-genai-gameserver.md

```diff
@@ -40,28 +40,12 @@ on that cluster, otherwise install Agones into your GenAI inference server clust
-server's endpoint. Also modify the `SIM_ENDPOINT` value to your inference server's endpoint.
-Alternatively you can create a basic http server that accepts requests in the structure noted in the
-above section, and returns a predetermined set of responses for the chat. The `GEN_AI_CONTEXT` is sent
-to the `GEN_AI_ENDPOINT` with each request, and the `SIM_CONTEXT` is sent to the `SIM_ENDPOINT` with
-each request as part of the GenAIRequest structure. The default values for `GEN_AI_CONTEXT` and
-`SIM_CONTEXT` are empty strings. The `PROMPT` is the first sent to prompt send to the GenAI endpoint
-to start the chat. The default values for the prompt is an empty string. `NUM_CHATS` is the number of
-requests made to the `SIM_ENDPOINT` and `GEN_AI_ENDPOINT`. The default value for is `NUM_CHATS` is `1`.
-
-If you want to set up the chat with the npc-chat-api from the [Google for Games GenAI](https://github.com/googleforgames/GenAI-quickstart/tree/main/genai/api/npc_chat_api)
-modify the [gameserver_npcchat.yaml](https://github.com/googleforgames/agones/blob/main/examples/simple-genai-server/gameserver_npcchat.yaml). Set either the
-`GEN_AI_ENDPOINT` or `SIM_ENDPOINT` to the NPC service `"http://genai-api.genai.svc/genai/npc_chat"`.
-Set whichever endpoint is pointing to the NPC service, either the `GEN_AI_NPC` or `SIM_NPC`,
-to be `"true"`. The `NPCRequest` to the NPC endpoint only sends the message (prompt), so any
-additional context outside of the prompt is ignored. `FROM_ID` is the entity sending messages to NPC,
-and `TO_ID` is the entity receiving the message (the NPC ID).
+To manually interact with `GenAiEndpoint` via netcat, change the `GEN_AI_ENDPOINT` value in the [gameserver_manualchat.yaml](https://github.com/googleforgames/agones/blob/main/examples/simple-genai-server/gameserver_manualchat.yaml) file to your inference server’s endpoint. Optionally, include the `GEN_AI_CONTEXT` in the same file that will include the context with each chat (post request) that you make to the GenAI endpoint.
+
+To make two clients "chat" to each other, change the `GEN_AI_ENDPOINT` and `SIM_ENDPOINT` values in the [gameserver_autochat.yaml](https://github.com/googleforgames/agones/blob/main/examples/simple-genai-server/gameserver_autochat.yaml) file to your inference server's address. You could also create a basic http server that receives requests in the structure noted in the above section, and returns a predetermined set of responses for the chat. When sending requests, `GEN_AI_CONTEXT` is sent
+to the `GEN_AI_ENDPOINT` and `SIM_CONTEXT` is sent to the `SIM_ENDPOINT` as part of the GenAIRequest structure. The default values for `GEN_AI_CONTEXT` and `SIM_CONTEXT` are empty strings. The conversation starts with a `PROMPT` that is sent to the GenAI endpoint, which is also an empty string by default. `NUM_CHATS` determines how many times messages are sent back and forth, with a default value of 1.
```

Comment:
We should make a note here that there is a rolling conversation, so after a conversation of length `NUM_CHATS` a new conversations starts. This continues until the game server is deleted with `kubectl delete`.

Reply:
Note that this is slightly different than the behavior with NPC Chat. In NPC Chat a new conversation is started either when it reaches the length `NUM_CHATS` or when is reaches the `STOP_PHRASE`, whichever comes sooner. As with the auto chat, this is a rolling set of conversations that continues until the game server is deleted.

Reply:
I've updated it as per your suggestion.